commons-hpcc/src/main/java/org/hpccsystems/commons/errors/HpccErrorCode.java

```diff
@@ -0,0 +1,20 @@
+package org.hpccsystems.commons.errors;
+
+public enum HpccErrorCode implements IErrorCode
+{
+
+    UNCAUGHT_EXCEPTION(200), INFO(300);

```

Comment:
Also, info sounds more like a err level

Reply:
An HpccError object takes an error level, error type and error code. In the case of info error level messages I have a basic "info" error code to use.